test/functional/feature_addrman.py

```diff
@@ -14,7 +14,9 @@
 from test_framework.test_node import ErrorMatch
 from test_framework.util import assert_equal
 
-
+ADDRMAN_NEW_BUCKET_COUNT = 1 << 10
+ADDRMAN_TRIED_BUCKET_COUNT = 1 << 8
+ADDRMAN_BUCKET_SIZE = 1 << 6
```

Comment:
nit:
space between constant declarations and functions

Reply:
pushed fix in [eb195d6](https://github.com/bitcoin/bitcoin/pull/28176/commits/eb195d66bcc1779b1385b5c0cd2e28dd081c1f0a)

Reply:
maybe we could move these globals into a common file? it's used in `test/functional/rpc_net.py` in #28523 too. (cc @0xB10C)

Reply:
> maybe we could move these globals into a common file? it's used in `test/functional/rpc_net.py` in #28523 too. (cc @0xB10C)

the best places I can think to make these values common are `test/functional/test_framework/util.py` or `test/functional/test_framework/netutil.py`